components/Form/index.js

```diff
@@ -0,0 +1,216 @@
+import { useRef, useState } from "react";
+import useSWR from "swr";
+import * as yup from "yup";
+
+const Form = ({ data, submit }) => {
+  const { data: issuesData, error: issuesError } = useSWR(`/api/issues`);
+  const [errors, setErrors] = useState([]);
+
+  // destructuring data obj
+  const {
+    title: contentTitle,
+    description: contentDescription,
+    content_type: contentType,
+    img_url: contentImgUrl,
+    video_url: contentVideoUrl,
+  } = data;
+
+  // for ref
+  const titleRef = useRef(null);
+  const descriptionRef = useRef(null);
+  const contentTypeRef = useRef(null);
+  const imgUrlRef = useRef(null);
+  const videoUrlRef = useRef(null);
+  const issueIdRef = useRef(null);
+
+  if (!issuesData && !issuesError) {
+    return <h1>Loading...</h1>;
+  }
+
+  if (!issuesData) {
+    return <h1>Not found</h1>;
+  }
+
+  const dataSchema = yup.object().shape({
+    title: yup.string().min(1).required(),
+    description: yup.string().required(),
+    content_type: yup.string().required(),
+    img_url: yup.string().url().required(),
+    video_url: yup.string().url(),
+  });
+
+  const handleSubmit = async (event) => {
+    event.preventDefault();
+
+    // data for page
+    const data = {
+      title: titleRef.current.value,
+      description: descriptionRef.current.value,
+      content_type: contentTypeRef.current.value,
+      img_url: imgUrlRef.current.value,
+      video_url: videoUrlRef.current.value,
+      relations: Number(issueIdRef.current.value),
+    };
+
+    try {
+      await dataSchema.validate(data, { abortEarly: false });
+      submit(data);
+    } catch (error) {
+      console.log(error.path);
+      console.log(error.inner);
```

Comment:
Do you definitely want to have both of these in? I'd guess you've been using them for troubleshooting.